tree/dataframe/src/RLoopManager.cxx

```diff
@@ -428,11 +431,12 @@ void RLoopManager::RunEmptySourceMT()
    ROOT::Internal::RSlotStack slotStack(fNSlots);
    // Working with an empty tree.
    // Evenly partition the entries according to fNSlots. Produce around 2 tasks per slot.
-   const auto nEntriesPerSlot = fNEmptyEntries / (fNSlots * 2);
-   auto remainder = fNEmptyEntries % (fNSlots * 2);
+   const auto nEmptyEntries = GetNEmptyEntries();
+   const auto nEntriesPerSlot = nEmptyEntries / (fNSlots * 2);
+   auto remainder = nEmptyEntries % (fNSlots * 2);
    std::vector<std::pair<ULong64_t, ULong64_t>> entryRanges;
    ULong64_t start = 0;
-   while (start < fNEmptyEntries) {
+   while (start < nEmptyEntries) {
       ULong64_t end = start + nEntriesPerSlot;
```

Comment:
There might be that possibility, but I don't know if that's for this PR. In this PR there are no user-visible changes, I just need to be able to internally change the range of entries that will be created in the single-core execution. So I just adapted the existing code to the fact there are now two numbers instead of one.

Reply:
On second thoughts, you are right in that we should be more consistent, because also the sequential execution case now offsets the begin and end entries according to the `fEmptyEntryRange` data member. I will change also the MT case